plugin-scalebar/src/main/java/com/mapbox/maps/plugin/scalebar/ScaleBarPluginImpl.kt

```diff
@@ -93,6 +97,16 @@ open class ScaleBarPluginImpl(
     mapCameraDelegate = delegateProvider.mapCameraDelegate
     mapListenerDelegate = delegateProvider.mapListenerDelegate
     mapTransformDelegate = delegateProvider.mapTransformDelegate
+    @Suppress("UNCHECKED_CAST")
+    cameraAnimationsPlugin = delegateProvider.mapPluginProviderDelegate.getPlugin(
+      Class.forName(
+        PLUGIN_CAMERA_ANIMATIONS_CLASS_NAME
+      ) as Class<CameraAnimationsPlugin>
+    )
+      ?: throw InvalidPluginConfigurationException(
+        "Can't look up an instance of plugin, " +
+          "is it available on the clazz path and loaded through the map?"
+      )
```

Comment:
This new dependency should be added to the README.md in the `ScaleBarPlugin`.

Reply:
I guess in this particular case(scale bar plugin), there will be no difference if we keep using the native camera listener, shall we keep using the native listeners here instead of introducing a new plugin dependency to the scalebar plugin?

Reply:
Reverted old listener.